javascript/javascript1/week4/voice_assistance.js

```diff
@@ -0,0 +1,135 @@
+let fName = "";
+const todos = [];
+let timer;
+function getReply(command) {
```

Comment:
from my perspective, the function is too long with the if else condition. try to split the function. maybe another function that can accept 'action and include the word' as a parameter.